lib/android/app/src/main/java/com/wix/reactnativenotifications/core/NotificationIntentAdapter.java

```diff
@@ -1,34 +1,54 @@
 package com.wix.reactnativenotifications.core;
 
 import android.annotation.SuppressLint;
+import android.app.Application;
 import android.app.PendingIntent;
-import android.app.TaskStackBuilder;
 import android.content.Context;
 import android.content.Intent;
+import android.os.Build;
 import android.os.Bundle;
 
+import com.wix.reactnativenotifications.Defs;
 import com.wix.reactnativenotifications.core.notification.PushNotificationProps;
 
 public class NotificationIntentAdapter {
     private static final String PUSH_NOTIFICATION_EXTRA_NAME = "pushNotification";
 
+    private Application application;
```

Comment:
are we using application anywhere in this file?

Reply:
done